src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -62,8 +62,15 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
         parser = argparse.ArgumentParser()
         parser.add_argument('--cerncastor', action='store_true')
         parser.add_argument('--old', action='store_true')
+        parser.add_argument('--disablewriterecovery', action='store_true')
```

Comment:
Why ? We don't have any old-style syntax in site-local-config to worry about for this parameter.